src/ui/FieldImage/FieldImage.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, Initialization } from '../../Core';
+import { exportGlobally } from '../../GlobalExports';
+import { IComponentBindings } from '../Base/ComponentBindings';
+import { IQueryResult } from '../../rest/QueryResult';
+import { FieldValue, IFieldValueOptions } from '../FieldValue/FieldValue';
+import { ComponentOptions, IFieldOption } from '../Base/ComponentOptions';
+
+export interface IFieldImage {
+  field?: IFieldOption;
+  width?: string;
+  height?: string;
+}
+
+/**
+ * The FieldImage component displays the value of a field associated to its parent search result.
+ * It is use strictly for displaying an image.
+ *
+ * This component is a result template component (see [Result Templates](https://developers.coveo.com/x/aIGfAQ)).
+ *
+ * A common use of this component is to display the image of a product for sale.
+ */
+export class FieldImage extends Component {
```

Comment:
So I asked @olamothe and we couldn't use the `thumbnail`.
From what I understood, if we used the thumbnail

- we would not be able to redirect to the product page of the image.
- we could have a login problem since the thumbnail uses it.
- thumbnail takes a lot of space in the index so it's better if we use it the least possible

if you want more details, you can ask @olamothe, it had something to do with the index but I don't really understand how it works.